core/templates/dev/head/css/oppia.css

```diff
@@ -3393,13 +3393,12 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-contributors-more-circle {
-  background: #888;
-  border-radius: 50%;
-  border-color: black;
+  background: grey;
```

Comment:
Can you use hex codes for both of these? I think we'd prefer to switch to that since it's more precise.